examples/default-values-backend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "backend_app_demo",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "yarn buildFrontend && tsc && node --insecure-http-parser built/src/index.js",
+    "buildFrontend": "parcel build public/index.html",
```

Comment:
Do we need a frontend portion? Maybe we can streamline the example a little bit more by getting rid of that

Reply:
I think it's nice to be able to see how a custom configuration page fits into the whole picture, especially since the other example (`page-location`) doesn't specify how to actually install the a frontend.
We don't mention it in the readme but if devs build on top of this code, it might be a great shortcut for them once they end up trying to build a frontend.

Reply:
Yeah I'm also in favour of keeping it. It was added at Frederick's request I think, as it makes the whole example more compelling